packages/server/src/course/ical.service.ts

```diff
@@ -27,7 +27,17 @@ export class IcalService {
         iCalElement.end !== undefined,
     );
 
-    return officeHours.map(event => ({
+    const isOfficeHoursEvent = (title: string) => {
+      const nonOfficeHourKeywords = ['Lecture', 'Lab', 'Exam', 'Class']
+      nonOfficeHourKeywords.forEach(keyword => {
```

Comment:
andmap reduces line 34 and line 37 omo

Reply:
`return nonOfficeHourKeywords.every(keyword => !title.includes(keyword));`

Reply:
oh sweet, ty!